cmake/P4CUtils.cmake

```diff
@@ -49,7 +49,8 @@ macro (p4c_add_library name symbol var)
 endmacro(p4c_add_library)
 
 # Add files with the appropriate path to the list of linted files
-function(add_cpplint_files dir filelist)
```

Comment:
Probably some warning comment is needed that these macros could be used by downstream projects so it is important to leave them as macros, etc..

Reply:
Now that we are adding cpplint files to a global property this might actually not be necessary any longer.